native/packages/react-native-bpk-component-button/stories.js

```diff
@@ -100,92 +101,109 @@ ArrowImage.defaultProps = {
   type: '',
 };
 
-const generateButtonStoryForType = type => (
-  <View key={type}>
-    <BpkText textStyle="sm" style={styles.bottomMargin}>Default</BpkText>
-    <View style={styles.btnContainer}>
-      <BpkButton
-        type={type}
-        title="Button"
-        onPress={action(`${type} pressed`)}
-        style={styles.buttonStyles}
-      />
-      <BpkButton
-        type={type}
-        selected
-        title="Selected"
-        onPress={action(`${type} selected pressed`)}
-        style={styles.buttonStyles}
-      />
-      <BpkButton
-        type={type}
-        disabled
-        title="Disabled"
-        onPress={action(`${type} disabled pressed, somehow`)}
-        style={styles.buttonStyles}
-      />
-      <BpkButton
-        type={type}
-        title="With icon"
-        icon={<ArrowImage type={type} />}
-        onPress={action(`${type} with icon clicked`)}
-        style={styles.buttonStyles}
-      />
-      <BpkButton
-        type={type}
-        title="Icon only"
-        icon={<ArrowImage type={type} />}
-        iconOnly
-        onPress={action(`${type} icon only button clicked`)}
-        style={styles.buttonStyles}
-      />
-    </View>
+// TODO nicer colors.
+const themeAttributes = {
+  gradientStartColor: '#CE93D8',
+  gradientEndColor: '#AB47BC',
+  textColor: 'rgba(255, 255, 255, 0.8)',
 
-    <BpkText textStyle="sm" style={styles.bottomMargin}>Large</BpkText>
-    <View style={styles.btnContainer}>
-      <BpkButton
-        large
-        type={type}
-        title="Button"
-        onPress={action(`${type} pressed`)}
-        style={styles.buttonStyles}
-      />
-      <BpkButton
-        large
-        type={type}
-        selected
-        title="Selected"
-        onPress={action(`${type} selected pressed`)}
-        style={styles.buttonStyles}
-      />
-      <BpkButton
-        large
-        type={type}
-        disabled
-        title="Disabled"
-        onPress={action(`${type} disabled pressed, somehow`)}
-        style={styles.buttonStyles}
-      />
-      <BpkButton
-        large
-        type={type}
-        title="With icon"
-        icon={<ArrowImage large type={type} />}
-        onPress={action(`${type} with icon clicked`)}
-        style={styles.buttonStyles}
-      />
-      <BpkButton
-        large
-        type={type}
-        title="Icon only"
-        icon={<ArrowImage large type={type} />}
-        iconOnly
-        onPress={action(`${type} icon only button clicked`)}
-        style={styles.buttonStyles}
-      />
+  selectedGradientStartColor: '#FF5722',
+  selectedGradientEndColor: '#E64A19',
+  selectedTextColor: '#FFEB3B',
+
+  disabledBackgroundColor: '#B2FF59',
+  disabledTextColor: '#b71c1c',
```

Comment:
See screenshot attached to PR for how these look. Right now I took them Material, but I anticipate that they'll be changed based on Design's feedback and possibly tokenised.